CHANGES.txt

```diff
@@ -1,3 +1,10 @@
+0.1
```

Comment:
@bduncan137 You could bump a minor version number and add your changes to the log.

Reply:
i thought about that, but figured I'd let you take all the credit so I could move on, but fine....I'll do a changelog.